libs/dh/core/shell/src/lib/dh-core-shell.component.spec.ts

```diff
@@ -37,6 +40,9 @@ describe(DhCoreShellComponent.name, () => {
       By.directive(WattShellComponent)
     );
 
+    expect(
+      screen.queryByRole('heading', { name: /datahub \[da\]/i })
+    ).toBeInTheDocument();
```

Comment:
nitpick: this is going to be removed very soon since the heading is not present in the prototype. Is it added to confirm that the testing module works?

Reply:
Yes it's only to ensure that the testing module actually works :)